CHANGELOG.rst

```diff
@@ -164,36 +159,24 @@ Changelog
   :doc:`/hazmat/backends/commoncrypto` and :doc:`/hazmat/backends/openssl`.
 * Added ``AES`` :class:`~cryptography.hazmat.primitives.ciphers.modes.CTR`
   support to the OpenSSL backend when linked against 0.9.8.
-* Added
-  :class:`~cryptography.hazmat.backends.interfaces.PKCS8SerializationBackend`
-  and
-  :class:`~cryptography.hazmat.backends.interfaces.TraditionalOpenSSLSerializationBackend`
-  support to the :doc:`/hazmat/backends/openssl`.
+* Added ``PKCS8SerializationBackend`` and
+  ``TraditionalOpenSSLSerializationBackend`` support to the
+  :doc:`/hazmat/backends/openssl`.
 * Added :doc:`/hazmat/primitives/asymmetric/ec` and
   :class:`~cryptography.hazmat.backends.interfaces.EllipticCurveBackend`.
 * Added :class:`~cryptography.hazmat.primitives.ciphers.modes.ECB` support
   for :class:`~cryptography.hazmat.primitives.ciphers.algorithms.TripleDES` on
   :doc:`/hazmat/backends/commoncrypto` and :doc:`/hazmat/backends/openssl`.
-* Deprecated
-  :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey` in
-  favor of backend specific providers of the
-  :class:`~cryptography.hazmat.primitives.interfaces.RSAPrivateKey` interface.
-* Deprecated
-  :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPublicKey` in favor
-  of backend specific providers of the
-  :class:`~cryptography.hazmat.primitives.interfaces.RSAPublicKey` interface.
-* Deprecated
-  :class:`~cryptography.hazmat.primitives.asymmetric.dsa.DSAPrivateKey` in
-  favor of backend specific providers of the
-  :class:`~cryptography.hazmat.primitives.interfaces.DSAPrivateKey` interface.
-* Deprecated
-  :class:`~cryptography.hazmat.primitives.asymmetric.dsa.DSAPublicKey` in favor
-  of backend specific providers of the
-  :class:`~cryptography.hazmat.primitives.interfaces.DSAPublicKey` interface.
-* Deprecated
-  :class:`~cryptography.hazmat.primitives.asymmetric.dsa.DSAParameters` in
-  favor of backend specific providers of the
-  :class:`~cryptography.hazmat.primitives.interfaces.DSAParameters` interface.
+* Deprecated the concrete ``RSAPrivateKey`` class in favor of backend
+  specific providers of the ``RSAPrivateKey`` interface.
```

Comment:
When we deprecated the concrete `cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey` class we said you should use the `cryptography.hazmat.primitives.interfaces.RSAPrivateKey` interface. We have subsequently moved that interface so it's located at... `cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey` (aka exactly where the concrete class used to live). Maybe the solution to this is to make the second definition point to `cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateKey` as the proper interface?